doc/ChangeLog

```diff
@@ -1,4 +1,57 @@
 ===============================================================
+Tag name: ctsm5.3.004
+Originator(s): samrabin (Sam Rabin)
+Date: Mon Oct  7 21:25:20 MDT 2024
+One-line Summary: Move hillslope data off surface datasets
+
+Purpose and description of changes
+----------------------------------
+
+Moves hillslope data off surface datasets onto its own separate hillslope_file. This makes it so we don't need to generate new surface datasets specifically for hillslope testing whenever the surface datasets are updated.
+
+
+Significant changes to scientifically-supported configurations
+--------------------------------------------------------------
+
+Does this tag change answers significantly for any of the following physics configurations?
+(Details of any changes will be given in the "Answer changes" section below.)
+
+    [Put an [X] in the box for any configuration with significant answer changes.]
+
+[ ] clm6_0
+
+[ ] clm5_1
+
+[ ] clm5_0
+
+[ ] ctsm5_0-nwp
+
+[ ] clm4_5
+
+
+Testing summary:
+----------------
+
+  regular tests (aux_clm: https://github.com/ESCOMP/CTSM/wiki/System-Testing-Guide#pre-merge-system-testing):
+
+    derecho ----- OK
+    izumi ------- OK
+
+
+Answer changes
+--------------
+
+Changes answers relative to baseline: Not in real runs, only our testing.
```

Comment:
@samsrabin I know this is merged, but could you elaborate briefly for my benefit?
Code configurations: I assume hillslope active. Other?
Platforms/compilers: I assume ALL.
Nature of change: ?
Definition of "real" runs: ?
Hypothesis for why in testing and not in real runs: ?

Reply:
Sorry, yes, only with hillslope active but on both Derecho and Izumi. It's only in our testing because I had to make changes to the test setup for compatibility; this included changing mesh file and hillslope data for some tests/testmods.